platform/terraform/02_kubernetes/terraform.tfvars

```diff
@@ -0,0 +1,35 @@
+###
+#
+# Authentication
+
+api_fingerprint      = "c6:1d:a5:54:c2:d2:67:26:6a:81:b8:e0:d2:f5:f7:a7"
+api_private_key_path = "/hoem/sdake/.oci/oci_api_key.pem"
+
+
+###
+#
+# Identity Management
+
+tenancy_id     = "ocid1.tenancy.oc1..aaaaaaaa6vyjrctvv5ax3lzuah3ldtlnrvni6hxcqdzcfoxjw5stgu4vz32q"
+compartment_id = "ocid1.compartment.oc1..aaaaaaaaq6xqdldlmtkmkpypkhsjymplonmuvbfpdqfii7ezu6b23utwqtba"
+user_id        = "ocid1.user.oc1..aaaaaaaa64i4tqgymgevje33u6tx7ejxgh2dipggg42lwikdr4f2ouwids5a"
+
+
+###
+#
+# Networking Configuration
+
+vcn_cidrs     = ["10.1.0.0/16"]
```

Comment:
I am not clear if vcn_cidrs needs to be set, or if it has sanitary defautls. I am also not sure why its a list.